rebench/rebench.py

```diff
@@ -256,23 +262,28 @@ def run(self, argv=None):
 
         runs = self._config.get_runs()
         does_profiling = any(r.is_profiling() for r in runs)
+        if self._config.options.disable_denoise:
+            return self.execute_experiment(runs, False, False)
```

Comment:
Hm, I think this doesn't do a bunch of things we normally do, including `init_environment(.)` and `.load_data`.

Perhaps we want to extract these bits that are not denoise-related into a separate method that we can call here?